packages/attendance_management/pubspec.lock

```diff
@@ -276,10 +276,11 @@ packages:
   dart_mappable_builder:
     dependency: "direct dev"
     description:
-      path: "../dart_mappable_builder"
-      relative: true
-    source: path
-    version: "4.2.0"
+      name: dart_mappable_builder
```

Comment:
This should be picked from local. Add a override file in each package which reads from local.